GVFS/GVFS.UnitTests/Common/GitObjectsTests.cs

```diff
@@ -0,0 +1,24 @@
+﻿using GVFS.Common.Git;
+using GVFS.Tests.Should;
+using NUnit.Framework;
+
+namespace GVFS.UnitTests.Common
+{
+    [TestFixture]
+    public class GitObjectsTests
+    {
+        [TestCase]
+        public void IsLooseObjectsDirectory()

```

Comment:
I would break this into 2 different test cases `IsLooseObjectsDirectory_ValidDirectories` and `IsLooseObjectsDirectory_InvalidDirectories` so a failure is more explicit about what didn't work.